docs/GettingStarted.md

```diff
@@ -12,8 +12,40 @@ your system, so that you can build apps with it right away. If you already
 have React Native installed, you can skip ahead to the
 [Tutorial](docs/tutorial.html).
 
-The instructions are a bit different depending on your development operating system, and whether you want to start developing for iOS or Android. If you
-want to develop for both iOS and Android, that's fine - you just have to pick
+## Quick Start
+
+[Create React Native App](https://github.com/react-community/create-react-native-app) is the easiest way to start building a new React Native application. It allows you to start a project without installing or configuring any native build tools.
+
+Assuming that you have Node installed, no Xcode or Android Studio installation is required:
```

Comment:
The phrasing here seems a bit odd to me (but keep in mind, English is my second language). The way I read it, Xcode or Android Studio may be required if I don't have Node installed.
Maybe we can split into two sentences, and swap the order? Something like:

> No Xcode or Android Studio installation is required. Assuming that you have Node installed, you can run the following commands to create a new React Native projected called "AwesomeProject":

I'm mirroring the instructions for Native Builds at the end here, "AwesomeProject" could be anything but ideally it would match the original RN init commands. 

After this PR gets merged I'll probably take another pass at updating the Getting Started guide and subsequent pages, as I'd like to update the tutorial so that it builds on top of the CRNA started project similar to what we did at F8 this year.